api/v1alpha1/networkvalidator_types.go

```diff
@@ -323,3 +361,20 @@ type NetworkValidatorList struct {
 func init() {
 	SchemeBuilder.Register(&NetworkValidator{}, &NetworkValidatorList{})
 }
+
+// HTTPFileAuthBytesDirect converts all of the inline basic authentication details in a NetworkValidatorSpec
+// to a map of rule names to basic auth details.
+func (s *NetworkValidatorSpec) HTTPFileAuthBytesDirect() map[string][]string {
```

Comment:
This function originally comes from validatorctl. Its been refactored to return a map[string][]string instead of a [][][]byte